package.json

```diff
@@ -24,9 +24,9 @@
     "@endo/eslint-plugin": "^0.4.4",
     "@endo/ses-ava": "^0.2.40",
     "@jessie.js/eslint-plugin": "^0.4.0",
-    "@typescript-eslint/eslint-plugin": "^7.1.0",
-    "@typescript-eslint/parser": "^7.1.0",
-    "ava": "^5.3.0",
+    "@typescript-eslint/eslint-plugin": "^7.8.0",
+    "@typescript-eslint/parser": "^7.8.0",
+    "ava": "^6.1.3",
```

Comment:
I'd like to try removing ava altogether since we don't have any ava test here anymore afaik